airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -408,10 +411,15 @@ private SyncCheckConnectionFailure checkConnections(final GenerateInputActivity.
     final StandardCheckConnectionInput sourceConfiguration = new StandardCheckConnectionInput().withConnectionConfiguration(sourceConfig);
     final CheckConnectionInput checkSourceInput = new CheckConnectionInput(jobRunConfig, sourceLauncherConfig, sourceConfiguration);
 
-    final JobCheckFailureInput jobStateInput =
-        new JobCheckFailureInput(Long.parseLong(jobRunConfig.getJobId()), jobRunConfig.getAttemptId().intValue(), connectionId);
-    var isLastJobOrAttemptFailure = jobCreationAndStatusUpdateActivity.isLastJobOrAttemptFailure(jobStateInput);
+    final int checkJobOutputVersion =
+        Workflow.getVersion(CHECK_PREVIOUS_JOB_OUTPUT_TAG, Workflow.DEFAULT_VERSION, CHECK_PREVIOUS_JOB_OUTPUT_TAG_CURRENT_VERSION);
+    boolean isLastJobOrAttemptFailure = true;
 
+    if (checkJobOutputVersion < CHECK_PREVIOUS_JOB_OUTPUT_TAG_CURRENT_VERSION) {
```

Comment:
This should be `if (checkJobOutputVersion >= CHECK_PREVIOUS_JOB_OUTPUT_TAG_CURRENT_VERSION)`

Reply:
changed it. thanks for noticing